models/orders.sql

```diff
@@ -0,0 +1,18 @@
+with payments as (
+    select * from {{ ref('stg_payments') }}
+),
+orders as (
+    select * from {{ ref('stg_orders') }}
+),
+final as (
+    select
+    payments.order_id,
+    orders.customer_id,
+    orders.order_date,
+    payments.amount,
+    payments.status
+from payments
+left join orders on payments.order_id = orders.order_id
```

Comment:
i would flip this and join payments to orders but that shouldn't functionally make much of a difference here, just a general best practice